e2e/malicious-join/malicious-join.go

```diff
@@ -172,7 +172,11 @@ func (j *maliciousJoiner) join(ctx context.Context) (*joinproto.IssueJoinTicketR
 		IsControlPlane:     false,
 	}
 	res, err := protoClient.IssueJoinTicket(ctx, req)
-	j.logger.Debug(fmt.Sprintf("Got join ticket response: %+v", res))
+	j.logger.Debug(fmt.Sprintf(
+		`Got join ticket response:
+  ApiServerEndpoint: %q
+  KubernetesVersion: %q
+`, res.ApiServerEndpoint, res.KubernetesVersion))
```

Comment:
```suggestion
	j.logger.Debug("Got join ticket response", "apiServerEndpoint", res.ApiServerEndpoint, "kubernetesVersion", res.KubernetesVersion)
```

should just use attributes